libafl_targets/src/cmplog.h

```diff
@@ -186,6 +190,7 @@ static inline void cmplog_routines_checked(uintptr_t k, const uint8_t *ptr1,
   hits &= CMPLOG_MAP_RTN_H - 1;
   MEMCPY(libafl_cmplog_map_ptr->vals.routines[k][hits].v0, ptr1, len);
   MEMCPY(libafl_cmplog_map_ptr->vals.routines[k][hits].v1, ptr2, len);
+  libafl_cmplog_map_ptr->vals.operands[k][hits].v0_is_const = 0;
```

Comment:
this callback is for routines. you don't need it here

Reply:
Again, this is basically an unused struct member - I still think it's worth making sure that it's initialised, even if it is always 0.

Reply:
hmm but this is a union. so you shouldn't be touching operand side of this struct.

Reply:
Good spot!